mbedtls/src/pk/dhparam.rs

```diff
@@ -18,6 +18,12 @@ define!(
     impl<'a> Into<ptr> {}
 );
 
+impl Into<*mut mbedtls_sys::dhm_context> for &Dhm {
```

Comment:
If you think this is useful-it's a bit odd conversion?-please implement this as part of the define macro.